lib/app.rb

```diff
@@ -3,18 +3,17 @@
 require_relative 'rental'
 require_relative 'student'
 require_relative 'teacher'
-require_relative 'modules/mod_people'
-require_relative 'modules/mod_books'
-require_relative 'modules/mod_rentals'
 
+require 'json'
+require 'fileutils'
 class App
   include PeopleMod
   include BooksMod
   include RentalsMod
```

Comment:
-  [ ] Kindly make sure that the app is working properly. This error appeared when I tried to run the app. You can fix this easily by requiring the modules called here.
![image](https://github.com/cancelei/school-library-CLI/assets/62290545/74744b50-5bd5-4da6-b2ce-83e161541f41)